pyrate/core/orbital.py

```diff
@@ -299,7 +299,7 @@ def __orb_inversion(design_matrix, data):
     B = design_matrix[~isnan(data)]
     d = data[~isnan(data)]
 
-    return dot(pinv(B, 1e-6), d)
+    return pinv(B) @ d
```

Comment:
Removing the artificial rcond restriction here allows us to solve the least-squares problem with more poorly-conditioned design matrices - this means we don't need to use a large "scale" factor in the tests any more.